cartography/graph/querybuilder.py

```diff
@@ -109,6 +109,27 @@ def _build_match_clause(matcher: TargetNodeMatcher) -> str:
     return ', '.join(match.safe_substitute(Key=key, PropRef=prop_ref) for key, prop_ref in matcher_asdict.items())
 
 
+def _build_where_clause_for_rel_match(node_var: str, matcher: TargetNodeMatcher) -> str:
```

Comment:
nonblocker: I'd rename this to something like `_build_where_clause_with_match_clause`